indexer/feature_utils.cpp

```diff
@@ -29,8 +30,29 @@ void GetMwmLangName(feature::RegionData const & regionData, StringUtf8Multilang
   }
 }
 
-void GetBestName(StringUtf8Multilang const & src, vector<int8_t> const & priorityList, string & out)
+bool GetTransliteratedName(feature::RegionData const & regionData, StringUtf8Multilang const & src, string & out)
 {
+  vector<int8_t> mwmLangCodes;
+  regionData.GetLanguages(mwmLangCodes);
+
+  string srcName;
+  for (auto const code : mwmLangCodes)
+  {
+    if (src.GetString(code, srcName))
+    {
+      out = Transliteration::GetInstance().Transliterate(srcName, code);
+      if (!out.empty())
+        return true;
+    }
+  }
+  if (!mwmLangCodes.empty() && src.GetString(StringUtf8Multilang::kDefaultCode, srcName))
+    out = Transliteration::GetInstance().Transliterate(srcName, mwmLangCodes[0]);
+  return !out.empty();
+}
+
+bool GetBestName(StringUtf8Multilang const & src, vector<int8_t> const & priorityList, string & out)
+{
+  out.clear();
```

Comment:
Same here, I suggest you to modify return parameter only when the function returns true.